tests/infra/is_snp.py

```diff
@@ -4,3 +4,5 @@
 import os
 
 IS_SNP = os.path.exists("/dev/sev")
+DEFAULT_SNP_SECURITY_POLICY_DIGEST = "ab6ccfea6e8c9c6233b1a4631cfd6440ecf910694b34d7e92e0236738758831e"
+DEFAULT_SNP_SECURITY_POLICY = "{\"allow_all\":true,\"containers\":{\"length\":0,\"elements\":null}}"
```

Comment:
I think we need a ticket and a TODO here to say that it'd be better to pick these up from the CI container directly (out of scope for this PR).